eli5/lightgbm.py

```diff
@@ -278,3 +358,30 @@ def _target_feature_weights(feature_weights_dict, num_features, bias_idx):
 
 def _get_score(feature_weights_dict):
     return sum(feature_weights_dict.values())
+
+
+def _get_shap_prediction_feature_weights(lgb, X, n_targets):
```

Comment:
I didn't dig into this function very much yet, but I have the intuition that using `np.reshape` function could simplify the code.

Reply:
@ivanprado Yes will make the required changes?

Reply:
Any progress in that?

Reply:
Yes, I changed that in the new commit. I will recheck